### PR TITLE
Consul releaser updates

### PR DESCRIPTION
**Pull request by joatmon08**
Simplified logic and set the releaser on fire when things fail.

### CODE REVIEW

File: src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java [hunk 1]
```diff
@@ -2,80 +2,215 @@
...
-    if (!world.isClient) {
-      Release release = Watcher.getRelease(entity.getApplication());
-      if (release != null) {
-        entity.status = release.Status;
-        entity.deploymentStatus = release.DeploymentStatus;
-        entity.traffic = release.CandidateTraffic;
-        entity.setPropertiesToState();
-        entity.sync();
-      }
+    public String getPrometheusAddress() {
```
Comment (eveld) [inline src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java:73]: Is this not the releaser address instead of prometheus?

Comment (joatmon08) [reply src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java:73]: No, there is additional prometheus address field that is passed through the configuration. For example:

```
  "monitor": {
    "plugin_name": "prometheus",
    "config": {
      "address": "http://localhost:9090",
      "queries": [
        {
          "name": "request-success",
          "preset": "envoy-request-success",
          "min":99
        },
        {
          "name": "request-duration",
          "preset": "envoy-request-duration",
          "min":2,
          "max": 200
        }
      ]
    }
  }
```

Comment (eveld) [reply src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java:75]: gotcha!

File: src/main/java/com/hashicorp/hashicraft/watcher/Watcher.java [hunk 2]
```diff
@@ -134,11 +135,11 @@ public static void Start(MinecraftServer server) {
   // }
   // }
 
-  public static HashMap<String, Release> getReleases() {
+  public static HashMap<String, ReleaseStatus> getReleases() {
     return releases;
   }
 
-  public static Release getRelease(String name) {
+  public static ReleaseStatus getRelease(String name) {
```
Comment (eveld) [inline src/main/java/com/hashicorp/hashicraft/watcher/Watcher.java:142]: Are we still using any of the functions in this file? Or can we get rid of the Watcher class entirely now?

File: src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java [hunk 3]
```diff
@@ -2,80 +2,217 @@
...
+    public String status = "";
+
+    @Syncable
+    public String deploymentStatus = "";
+
+    @Syncable
+    public Integer traffic = 0;
+
+    private ExecutorService executor;
+    private Releaser releaser = new Releaser(address);
```
[the remark this answers is not in the document]
Comment (joatmon08) [reply src/main/java/com/hashicorp/hashicraft/block/entity/ConsulReleaserEntity.java:47]: I think it's a nice to-do, we'll leave it for now though. I suspect we have more logic to add / refactor anyway.

**Pull request merged**